bgpd/bgp_route.c

```diff
@@ -4501,6 +4506,91 @@ static void bgp_soft_reconfig_table(struct peer *peer, afi_t afi, safi_t safi,
 				return;
 			}
 		}
+	}
+}
+
+int bgp_soft_reconfig_table_index_sequencer(struct thread *thread)
+{
+	struct soft_reconfig_table_attr *attr = THREAD_ARG(thread);
```

Comment:
Let's call this variable something else besides attr.  attr is specifically for `struct attr *` that we use everywhere in bgp.  This is not a `struct attr *`